TimePlatformer/Assets/Scripts/EnemyHorizontalMovementScript.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class EnemyHorizontalMovementScript : MonoBehaviour {
+
+    [Range(-1, 1)] public int direction;
```

Comment:
Why is this a range, it seems like this should be a boolean or enumeration since there are a finite number of directions?